Pods/EPContactsPicker.swift

```diff
@@ -233,6 +254,75 @@ open class EPContactsPicker: UITableViewController, UISearchResultsUpdating, UIS
             CNContactEmailAddressesKey as CNKeyDescriptor,
         ]
     }
+  
+  fileprivate func design(textField: UITextField, placeholderText: String) {
+    textField.attributedPlaceholder = NSAttributedString(string: placeholderText,
+                                                         attributes: [NSForegroundColorAttributeName: UIColor.black])
+    textField.font = UIFont(name: "HelveticaNeue", size: 14.0)!
+    let heightConstraint = NSLayoutConstraint(item: textField, attribute: .height, relatedBy: .equal,
+                                              toItem: nil, attribute: .notAnAttribute, multiplier: 1, constant: 25)
+    textField.addConstraint(heightConstraint)
+  }
+  
+  fileprivate func addContactAction(_ alertController: UIAlertController) {
+    let nameField = (alertController.textFields![0] as UITextField).text ?? ""
+    let phoneField = (alertController.textFields![1] as UITextField).text ?? ""
+    
+    if nameField != "" && phoneField != "" {
```

Comment:
I would: 
(1) remove " ?? "" " from the test fields extraction 
(2) use guard to see if they're nil